x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -77,6 +78,23 @@ const executeTimeoutValidator = (argData: ParsedArgData): true | string => {
   }
 };
 
+const filePathValidator = (argData: ParsedArgData): true | string => {
+  const emptyResult = emptyArgumentValidator(argData);
+  if (emptyResult !== true) {
+    return emptyResult;
+  } else if (
+    String(argData) &&
+    String(argData).trim().length > 0 &&
+    /\/|\\/.test(String(argData))
```

Comment:
Can you explain what this is attempting to validate? (also maybe add a comment describing that).

Looks like its making sure that the path must have at least 1 forward or backslash... Are you sure we want to do that?

I think its possible to have paths that don't actually have any slashes - like network shares? or perhaps use of env. variables? (not sure that is something that is supported)... Like `scan --path="%ProgramFiles%"`? or on *nix systems could one not use `~` for home directory?

Reply:
Yeah I went with checking a slash `/` (mac/linux) or a backslash `\` for windows. Indeed we might also have `~`. We should have some validation here for sure that checks some of these conditions I think.

Reply:
I don't think you should include any validation for `--path` outside of "its not blank". I remember discussing this too when we implemented `get-file` and determine that there could be too many cases to cover and we may not get it right, so we just accept the user's input and let the Endpoint error out if the OS can't support the provided value.

that being said - you should check with the endpoint team if env. variables can be used or even network shares... not that we would build validation for them, but that would be good info. to add to our documentation for this command.

Reply:
99e13666cb030e03427e48b3bf935b09b293207d